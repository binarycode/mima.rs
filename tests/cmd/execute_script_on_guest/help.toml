args = 'help execute-script-on-guest'
status.code = 0
fs.sandbox = true
