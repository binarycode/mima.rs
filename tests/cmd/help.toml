args = 'help'
status.code = 0
fs.sandbox = true
