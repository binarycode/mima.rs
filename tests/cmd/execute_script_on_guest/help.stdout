Execute script on guest

Usage: mima execute-script-on-guest <GUEST_ID> [PATH] [-- [ARGS]...]

Arguments:
  <GUEST_ID>  Guest ID
  [PATH]      File path
  [ARGS]...   Arguments to pass to the file

Options:
  -h, --help  Print help
